docs/testing/test-debug-office-add-ins.md

```diff
@@ -17,9 +17,17 @@ Office Add-ins run across major platforms, so you need to test an add-in in all
 > On Windows computers, the version of Windows and Office will determine which browser control is used by add-ins. For more information, see [Browsers used by Office Add-ins](../concepts/browsers-used-by-office-web-add-ins.md).
 
 > [!IMPORTANT]
```

Comment:
This IMPORTANT note is getting huge. Should it become an H2 section?

Reply:
I was going back and forth on this. Now that I look at it again, the note does take up majority of the article space now. I'll move this into its own section.